cloudsync/tasks_test.py

```diff
@@ -441,6 +470,32 @@ def test_update_youtube_statuses(mocker):
     assert YouTubeVideo.objects.filter(status=YouTubeStatus.PROCESSED).count() == 5
 
 
+@override_settings(ENABLE_VIDEO_PERMISSIONS=True)
+def test_update_youtube_statuses_quota_exceeded(mocker):
+    """
+    Test that the update_youtube_statuses task stops without raising an error if the quota is exceeded.
```

Comment:
SPECIFICITY: perhaps "...if the quota is exceeded" => "... if the *api* quota is exceeded"

Reply:
will do